cmd/juju/commands/main_test.go

```diff
@@ -645,6 +645,7 @@ func (s *MainSuite) TestHelpCommands(c *gc.C) {
 	cmdSet := set.NewStrings(commandNames...)
 	if !featureflag.Enabled(feature.JujuV3) {
 		cmdSet.Add("run-action")
+		cmdSet.Add("run")
```

Comment:
"call"

Reply:
"run" is what's needed here.
If feature flag is *not* enabled, there is a run command